HEC.FDA.Statistics/Distributions/Empirical.cs

```diff
@@ -431,11 +447,11 @@ public static double Subtract(double x1, double x2)
         {
             return x1 - x2;
         }
-       
+
 
         public XElement WriteToXML()
         {
-            XElement masterElem = new("Empirical Distribution");
+            XElement masterElem = new("Empirical_Distribution");
```

Comment:
ensure we're reading this correctly now

Reply:
we don't actually read and write empirical in FDA. Rename is safe. There can't be any in the wild to migrate forward if it was never hooked to a control.